buildscripts/condarecipe_clone_icc_rt/meta.yaml

```diff
@@ -1,31 +1,35 @@
-{% set version = "2018.0.2" %} # this is the intel version to get
-{% set build_number = "0" %}   # and the build number from the intel version
+{% set version = "2019.3" %} # this is the intel version to get
+{% set win_build_number = "203" %}   # the build number from the intel windows version
+{% set osx_build_number = "199" %}   # the build number from the intel osx version
+{% set lnx_build_number = "199" %}   # the build number from the intel linux version
 
 package:
   name: icc_rt
   version: {{ version }}
 
 build:
-  number: {{ build_number }}
+  number: {{ win_build_number }}  # [win]
+  number: {{ osx_build_number }}  # [osx]
+  number: {{ lnx_build_number }}  # [linux]
 
 source:
-    - url: https://anaconda.org/intel/icc_rt/{{ version }}/download/win-64/icc_rt-{{version}}-intel_{{build_number}}.tar.bz2 # [win]
-    - md5: 8ba0b0d3fbffdb0cd9febbf90752c4ad # [win]
-    - sha256: e2fbb6452d6544325fa2548c11c7a0989083ce6240c64e83963cf44e564a10dc # [win]
+    - url: https://anaconda.org/intel/icc_rt/{{ version }}/download/win-64/icc_rt-{{version}}-intel_{{win_build_number}}.tar.bz2 # [win]
+    - md5: d39bae3218457a4ea045763fdcfc1562 # [win]
+    - sha256: 2c55b8af1dea35ee4648b671050899a93b7eba1b26acad019bf569ca777a944e # [win]
 
-    - url: https://anaconda.org/intel/icc_rt/{{ version }}/download/osx-64/icc_rt-{{version}}-intel_{{build_number}}.tar.bz2 # [osx]
-    - md5: 6c6c59bd819785dfd281f3107ad39ad0 # [osx]
-    - sha256: 292334590092c973737eabf138dd28f75542054cef7ce04d762de4087504b8c2 # [osx]
+    - url: https://anaconda.org/intel/icc_rt/{{ version }}/download/osx-64/icc_rt-{{version}}-intel_{{osx_build_number}}.tar.bz2 # [osx]
+    - md5: 064566ac53e729d3f008e32b1f73d1fa # [osx]
+    - sha256: 54a372b0d8d5b4d750c28ea122851b52ec9aa3cccb8d4cf4a2999494dfda6656 # [osx]
 
-    - url: https://anaconda.org/intel/icc_rt/{{ version }}/download/linux-64/icc_rt-{{version}}-intel_{{build_number}}.tar.bz2 # [linux]
-    - md5: b814ae46604a6ef3f140f00c3f0b9d25 # [linux]
-    - sha256: 68d8097b340ed8c876101590a5c1b69beb302491439b9771bedeed3c8012a520 # [linux]
+    - url: https://anaconda.org/intel/icc_rt/{{ version }}/download/linux-64/icc_rt-{{version}}-intel_{{lnx_build_number}}.tar.bz2 # [linux]
+    - md5: 306c3ee9491577715dbd76c838147078 # [linux]
+    - sha256: 4cedd10343d1ab4403af2ff080b47afe5399be550f1c215e5a7c7eceec672516 # [linux]
 
     - path: scripts
 
 requirements:
   build:
-    - python>=3.6
```

Comment:
Why was the requirement >= 3.6 before?

Reply:
Looks like there is no obvious explanation, so we should remove this requirement since this is a non-Python library anyway.